modules/new_profile/php/new_profile.class.inc

```diff
@@ -310,14 +306,6 @@ class New_Profile extends \NDB_Form
             }
         }
 
-        // Validate site entered
```

Comment:
you cant entirely delete the site validation

Reply:
I think we need the date of birth stuff too.

Reply:
> you cant 
I read `cant` as `can`, in fact, this check is unnecessary.